sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/autorest.md

```diff
@@ -16,7 +16,6 @@ batch:
 - input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/b791f57426508cb2793a8911650a416dcb11c6a6/specification/cognitiveservices/data-plane/Language/stable/2021-10-01/questionanswering-authoring.json
 # namespace: Azure.AI.Language.QuestionAnswering.Projects
   add-credentials: true
-  data-plane: true
```

Comment:
Can you describe in your PR description what the effect of these changes should be instead of the standard external contributor template? For example, what was the intended impact of his directive given that there was seemingly no other effect on sources? And shy isn't the `generation1-convenience-client` directive added given that this swagger is meant to generate an LLC (the old style)?

I don't feel I have enough information to sign off given we're about to GA this SDK, and are about to release a beta of another one I don't see modified here that is also an older LLC (we'll use DPG for beta 4).

Reply:
I'll take this offline to unblock the PR since there's no source changes, though I don't understand why there aren't any changes besides this.

Reply:
We will change to set 'data-plane' as a default target in autorest.csharp codegen. And previous HLC library will use a new tag 'generation1-convenience-client'. So when 'azure-arm' is set, it will generate management plane, if 'generation1-convenience-client' is set, it will generate HLC, otherwise it will generate DPG(LLC). 

The issue for this is https://github.com/Azure/azure-sdk-for-net/issues/assigned/chunyu3